commit 17cc7fe6efbb6999adfda60e47732ad715a03eed
Author: Takashi Iwai <tiwai@suse.de>
Date:   Tue Sep 4 17:58:42 2018 +0200

    staging: bcm2835-audio: Drop useless NULL check
    
    commit 8bcf9f252c29c2d5bcce3db605c0ebf1ef230f9c upstream.
    
    alsa_stream->chip can be never NULL.
    
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Tested-by: Stefan Wahren <stefan.wahren@i2se.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c b/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c
index c935c6e99633..13c61af4e723 100644
--- a/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c
+++ b/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c
@@ -188,8 +188,7 @@ static int snd_bcm2835_playback_close(struct snd_pcm_substream *substream)
 	alsa_stream->buffer_size = 0;
 
 	bcm2835_audio_close(alsa_stream);
-	if (alsa_stream->chip)
-		alsa_stream->chip->alsa_stream[alsa_stream->idx] = NULL;
+	alsa_stream->chip->alsa_stream[alsa_stream->idx] = NULL;
 	/*
 	 * Do not free up alsa_stream here, it will be freed up by
 	 * runtime->private_free callback we registered in *_open above
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1453_17cc7fe6efbb6999adfda60e47732ad715a03eed.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1453_17cc7fe6efbb6999adfda60e47732ad715a03eed.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
